pyfive/dataobjects.py

```diff
@@ -230,11 +231,17 @@ def _parse_attribute_msg(self, buffer, offset):
 
         # Read the dataspace information
         shape, maxshape = determine_data_shape(buffer, offset)
-        items = int(np.prod(shape))
-        offset += _padded_size(attr_dict['dataspace_size'], padding_multiple)
 
-        # Read the value(s)
-        value = self._attr_value(dtype, buffer, items, offset)
+        # detect Empty/NULL dataspace
+        if shape is None:
+            value = Empty(dtype=dtype)
```

Comment:
After detecting NULL dataspace in `determine_data_shape`, we create Empty value here.

Reply:
It'd be good to fix the docstring for empty, as we're now actually using it!

Reply:
Done!